app/src/main/java/com/duckduckgo/app/trackerdetection/blocklist/BlockList.kt

```diff
@@ -99,6 +105,23 @@ class BlockListPixelsPlugin @Inject constructor(private val inventory: FeatureTo
     }
 }
 
+@ContributesMultibinding(AppScope::class)
+class BlockListPrivacyConfigCallbackPlugin @Inject constructor(
+    private val inventory: FeatureTogglesInventory,
+    private val trackerDataDownloader: TrackerDataDownloader,
+    @AppCoroutineScope private val coroutineScope: CoroutineScope,
+    private val experimentAA: ExperimentTestAA,
+) : PrivacyConfigCallbackPlugin {
+    override fun onPrivacyConfigDownloaded() {
+        experimentAA.experimentTestAA().isEnabled(CONTROL)
```

Comment:
we're calling isEnabled without checking any value, why?
If we need to call it, shouldn't it be inside the `launch`